deployments/liqo/crds/sharing.liqo.io_advertisements.yaml

```diff
@@ -14,8 +14,6 @@ spec:
     listKind: AdvertisementList
     plural: advertisements
     singular: advertisement
-    shortNames:
-      - adv
```

Comment:
You removed the shortName of the Advertisement, I guess it was unintentional.

Reply:
Actually, I removed this as the outcome of the run, since not present when generated from scratch.
Probably, the corresponding kubebuilder annotation had been removed from the API definition, but the CDR was not updated

Reply:
I confirm the same happened to me a few days ago. Nevertheless I just checked and the annotation is still present